iina/MainWindowController.swift

```diff
@@ -2114,7 +2114,7 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
     if player.info.isNetworkResource {
       bufferIndicatorView.isHidden = false
       bufferSpin.startAnimation(nil)
-      bufferProgressLabel.stringValue = "Opening stream..."
+      bufferProgressLabel.stringValue = NSLocalizedString("main.opening_stream", comment:"Opening stream…")
```

Comment:
This is kind of a breaking change to all translators. Since some of the translators have finished beta 4 work, we should avoid such braking changes. You can revert it and make it another pr. I'll consider merge after beta 4 is released.

Reply:
Sorry, I wasn’t aware this would be an issue, I thought you’d add the English wording for missing strings before release anyway?
But if that’s a problem, feel free to merge this PR after release, there’s no hurry on my side.

Reply:
I mean you could revert just these two line and move them into another PR. Other parts of this PR are good. And I want to update German localization in beta 4 since German localization has not been updated for quite a while.

Reply:
Alright then, done.